travis-results-2.txt

```diff
@@ -0,0 +1,204 @@
+Transforming the AsciiDoc content to DocBook XML...
+>>> Finished with service_mesh book in openshift-enterprise <<<
+---------------------------------------
+>>> Working on metering book in openshift-enterprise <<<
+Transforming the AsciiDoc content to DocBook XML...
+>>> Finished with metering book in openshift-enterprise <<<
+---------------------------------------
+>>> Working on web_console book in openshift-enterprise <<<
+Transforming the AsciiDoc content to DocBook XML...
+>>> Finished with web_console book in openshift-enterprise <<<
+---------------------------------------
+>>> Working on monitoring book in openshift-enterprise <<<
+Transforming the AsciiDoc content to DocBook XML...
+>>> Finished with monitoring book in openshift-enterprise <<<
+---------------------------------------
+>>> Working on authentication book in openshift-enterprise <<<
+Transforming the AsciiDoc content to DocBook XML...
+>>> Finished with authentication book in openshift-enterprise <<<
+---------------------------------------
+>>> Working on logging book in openshift-enterprise <<<
+Transforming the AsciiDoc content to DocBook XML...
+asciidoctor: WARNING: includes/cluster-logging-deploy-console.adoc: line 178: callout list item index: expected 7, got 6
+asciidoctor: WARNING: includes/cluster-logging-deploy-console.adoc: line 179: callout list item index: expected 8, got 7
+asciidoctor: WARNING: includes/cluster-logging-deploy-console.adoc: line 180: callout list item index: expected 9, got 8
+asciidoctor: WARNING: includes/cluster-logging-deploy-console.adoc: line 181: callout list item index: expected 10, got 9
+asciidoctor: WARNING: includes/cluster-logging-deploy-console.adoc: line 182: callout list item index: expected 11, got 10
+asciidoctor: WARNING: includes/cluster-logging-deploy-console.adoc: line 183: callout list item index: expected 12, got 11
+asciidoctor: WARNING: includes/cluster-logging-deploy-console.adoc: line 183: no callout found for <12>
+>>> Finished with logging book in openshift-enterprise <<<
+---------------------------------------
+>>> Working on scalability_and_performance book in openshift-enterprise <<<
+Transforming the AsciiDoc content to DocBook XML...
+>>> Finished with scalability_and_performance book in openshift-enterprise <<<
+---------------------------------------
+>>> Working on cli_reference book in openshift-enterprise <<<
+Transforming the AsciiDoc content to DocBook XML...
+>>> Finished with cli_reference book in openshift-enterprise <<<
+---------------------------------------
+>>> Working on backup_and_restore book in openshift-enterprise <<<
+Transforming the AsciiDoc content to DocBook XML...
+asciidoctor: WARNING: includes/dr-recover-expired-control-plane-certs.adoc: line 36: unterminated listing block
+[31m[1mattributes construct error, line 110, column 19 (master.xml, line 110)[0m
+>>> Finished with backup_and_restore book in openshift-enterprise <<<
```

Comment:
Caught after second command, `python3 makeBuild.py >> travis-results-2.txt`, is ran.

Kind of confusing message here, bc the warning is _not_ located in `dr-recover-expired-control-plane-certs.adoc`, and the part that you need to pay attention to is line `202` which looks like it's a part of the previous line.

Might be a good idea to add to the instructions that if `master.xml` is present anywhere in the output, users should look in that file (located in `./drupal-build/openshift-enterprise/backup_and_restore/build/`) to figure out where the issue is occurring.

This error is for the missing quote in an ID in one of the test modules.